content/22.vaccines.md

```diff
@@ -669,9 +678,14 @@ Additionally, Germany has stated that if Sputnik V were approved by EMA, it woul
 Russia is developing other vaccine candidates and has approved a third vaccine, CoviVac, which is an inactivated vaccine produced by the Chumakov Centre in Moscow, despite the fact the clinical trials have yet to begin [@url:https://www.reuters.com/article/us-health-coronavirus-russia-vaccine-idUSKBN2AK07H].
 
 In Asia, China and India are the main COVID-19 vaccination developers and providers.
+Thus far, over 63 million people have been vaccinated against COVID-19 vaccine in India [@url:https://www.mygov.in/covid-19/].
+A broad range of COVID-19 vaccine candidates are under investigation in order to respond to the COVID-19 pandemic.
 In India, the Covaxin vaccine produced by Bharat Biotech received emergency authorization on January 3rd, 2021, despite the lack of phase III data until March 3rd [@url:india-covid-19-vaccine.html; @url:https://www.bharatbiotech.com/images/press/covaxin-phase3-efficacy-results.pdf].
 Following the release of the phase III data indicating 81% efficacy, Zimbabwe authorized the use of Covaxin [@url:http://www.xinhuanet.com/english/2021-03/04/c_139783893.htm].
-In February, 2021, Bharat Biotech received approval from Indian officials to commence a phase I study of an intranasal chimpanzee-adenovirus (ChAd) vectored SARS-CoV-2-S vaccine called BBV154 [@url:https://www.bharatbiotech.com/intranasal-vaccine.html].
+In addition to COVAXIN®, Bharat Biotech has also developed an adenovirus vector-based intranasal BB154 vaccine candidate that induces neutralizing IgG, mucosal IgA, and T cell responses [@url:https://www.bharatbiotech.com/intranasal-vaccine.html]. 
+In February, 2021, Bharat Biotech received approval from Indian officials to commence a phase I study of an intranasal chimpanzee-adenovirus (ChAd) vectored SARS-CoV-2-S vaccine called BBV154 [@url:https://www.bharatbiotech.com/intranasal-vaccine.html]. 
```

Comment:
If this is the same vaccine as the line above, maybe just rephrase slightly to make that more clear? like "their" instead of "an" before "intranasal chimpanzee-adenovirus vectored"

Reply:
I am new to GitHub. Struggling to get back to the original content and edit this or give my information as you requeste. Sorry about this. Thanks for your patience.

Reply:
I absolutely understand @NAIKA86, it's quick the learning curve! You can use [these directions](https://github.com/greenelab/covid19-review/blob/master/CONTRIBUTING.md#the-how-to-of-review) if you want, they might make it a little easier!

Reply:
For the author info, you can just put it in a comment and I can put it into the file, if that's easier!

Reply:
github: NAIKA86
name: Amruta Naik
initials: AN
orcid:https://orcid.org/0000-0003-0673-2643
twitter: (optional)
email: NAIKA@chop.edu
Do you agree to the code of conduct? Yes
affiliation(s): Children's Hospital of Philadelphia
Do you have any conflicts of interest? No
Do you have any funding you'd like listed? (And what is it?) No

Reply:
Awesome! It's updated!

Reply:
```suggestion
In February, 2021, Bharat Biotech received approval from Indian officials to commence a phase I study of this intranasal chimpanzee-adenovirus (ChAd) vectored SARS-CoV-2-S vaccine [@url:https://www.bharatbiotech.com/intranasal-vaccine.html]. 
```